src/core/fetch.ts

```diff
@@ -1,6 +1,6 @@
 import { serialize, deserialize } from '@/core/units/parseObject';
 
-const host = '/api';
+const host = process.env.VUE_APP_API_URI;
```

Comment:
I feel like adding `/datastore` here makes more sense than in the api calls in the other places of the source.

Makes it less redundant and less of a pain if `datastore` were to change at some point.

Reply:
For `fetch.ts`, host refers to the entire backend - not just the datastore.

Probably need some semi-dynamic config somewhere. "datastore" and "history" are very stable, but each controller also has its own subdomain.